src/util/test_common.rs

```diff
@@ -74,20 +73,18 @@ pub fn random_bools(n: usize) -> Vec<bool> {
 }
 
 pub fn random_numbers<T: Rand>(n: usize) -> Vec<T> {
-  let mut result = vec!();
+  let mut result = vec![];
   let mut rng = thread_rng();
   for _ in 0..n {
     result.push(rng.gen::<T>());
   }
   result
 }
 
-pub fn random_numbers_range<T>(
-  n: usize,
-  low: T,
-  high: T,
-  result: &mut Vec<T>
-) where T: PartialOrd + SampleRange + Copy {
+pub fn random_numbers_range<T>(n: usize, low: T, high: T, result: &mut Vec<T>)
```

Comment:
I like the old format better (even though the parameters could be put in a single line).

Reply:
Unfortunately, I do not know how to enforce formatting of function arguments, apart from using `#[cfg_attr(rustfmt, rustfmt_skip)]` for each item.